pipenv/cli.py

```diff
@@ -209,7 +209,7 @@ def do_install_dependencies(dev=False, only=False, bare=False, requirements=Fals
     else:
         if not bare:
             click.echo(crayons.yellow('Installing dependencies from Pipfile.lock...'))
-        with open(project.lockfile_location, 'r') as f:
+        with open(project.lockfile_location) as f:
```

Comment:
Also according to Python documentation at https://docs.python.org/3/library/functions.html#open `It defaults to 'r' which means open for reading in text mode.`